src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj

```diff
@@ -9,6 +9,13 @@
     <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.KeyVault" Version="2.0.6" />
```

Comment:
@jaredmoo I was under the impression Nathan Fan updated all SQL Tests to use latest Resource Manager nuget.
I see you are using a really old resource manager to record your tests, please update to the latest (>=1.6.0)